### ISSUE TITLE
Storytlr as a Wordpress Plugin to embed your lifestream in your blog

### ISSUE THREAD

**Issue by saltedlolly** [eschnou/storytlr#41]
I have been looking for a way to create a Lifestream page in my Wordpress install. Most of the existing plugins in this space seem to have stagnated. How feasible is that Storytlr could be converted into a Wordpress plugin?

**Comment by jmhobbs** [comment]
I don't think that the whole Storytlr install could be wrapped into Wordpress, but it might be feasible to access the main view through a plugin somehow, or probably just a wrapper that reads the same database.

**Comment by vbk** [comment]
Jmhobbs, 

I went trough other requests for WordPress integration before I post this. There are a couple of plugins with similar functionality in WordPress repository , but they are not well featured like Storytlr. Infact, Storytlr would be more comfortable as separate installation as well as integrated module in WordPress. 

I am very much eager to have an option to integrate in WordPress. Hope there would be a plugin be released in near future 

Thank you!
